docs/additional-functionality/qualification-profiling-tools.md

```diff
@@ -219,6 +219,19 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
                                     only logs for which all conditions are satisfied.
                                     Example: <Filter1> <Filter2> <Filter3> --all -> result
                                     is <Filter1> AND <Filter2> AND <Filter3>. Default is all=true.
+  -c, --configuration-names <arg>   Filter applications based on certain configurations that were
+                                    set during launch of the application. It can filtered based
+                                    on specific values or any values for a config. Multiple configs
+                                    can be provided where the filtering is done if any of the
+                                    config is present in the eventlog. Specific configurations
+                                    takes precedence over at all configurations.
+                                    filter on specific configuration:
+                                    --configuration-names=spark.eventLog.enabled:true
+                                    filter all eventlogs which has config:
+                                    --configuration-names:spark.driver.port
```

Comment:
I had it `=` initially but changed it `:` for better readability. 
 If we are providing keys and values, it would be something like:
` --configuration-names=spark.eventLog.enabled=true` which may be confusing.  Will change it back to `=`

Reply:
that isn't what I meant

Reply:
--configuration-names:spark.driver.port that should be  --configuration-names=spark.driver.port right?

Reply:
Sorry, got confused.  Will change it here.